include/dali/c_api.h

```diff
@@ -33,7 +33,8 @@ extern "C" {
 typedef struct {
   void *pipe;
   void *ws;
-  void *batch_size_map;     /// @see batch_size_map_t
+  void *batch_size_map;      /// @see batch_size_map_t
+  void *data_id_map;         /// @see data_id_map_t
```

Comment:
Move the new field to the end of the structure.

Reply:
Done.

Reply:
Nope?

Reply:
It is now. Sorry for that.